File: C++/009_Palindrome_Number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        string s = to_string(x) ;
        string tmp = s ;
        reverse(s.begin(),s.end()) ;
        if (strcmp(s.c_str(),tmp.c_str()) == 0 )
            return true ;
        return false ;
    }
};

File: Python/009_Palindrome_Number.py
class Solution(object):
    def isPalindrome(self, x):
        str_x = str(x)
        max_index = len(str_x)
        for i in range(max_index/2) :
            temp = max_index - 1 - i
            if str_x[i] != str_x[temp] :
                return False
        return True

